Firestore/Example/Tests/Remote/FSTSerializerBetaTests.mm

```diff
@@ -104,18 +106,7 @@ bool IsWatchChangeEqual(const WatchChange &lhs, const WatchChange &rhs) {
 NS_ASSUME_NONNULL_BEGIN
 
 @interface FSTSerializerBeta (Test)
-- (GCFSValue *)encodedNull;
```

Comment:
Note that I'm making these public in the `FSTSerializerBeta` interface rather than continuing this pattern because this made for some enormously difficult to debug test failures when I changed the signature of the implementation but failed to update callers using these declarations. The compiler doesn't enforce that these match and so the caller would pass an `NSString*` and the callee would interpret that pointer and adjacent bytes on the stack it as an `absl::string_value`.